modules/tests/src/test/scala/lucuma/itc/service/GraphQLIntTimeErrorsSuite.scala

```diff
@@ -94,13 +94,13 @@ class GraphQLIntTimeErrorsSuite extends FailingCalculationSuite {
       json"""
         {
           "errors": [{
-            "message": "A calculation error",
+            "message": "Error calculating ITC: A calculation error",
```

Comment:
I think this may look odd on the frontend, or at least require some changes in explore

Reply:
Explore is now showing error messages from old-old untouched, which causes confusion as to where they originate. This change aims to make that more clear.